independent-publisher-connectors/Safaricom Mpesa Daraja/apiProperties.json

```diff
@@ -0,0 +1,49 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "4b048dda-9dc5-412b-b881-1f6881f882d8",
+          "scopes": [
+            "User.Read"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://login.microsoftonline.com/common/oauth2/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://daraja.onrender.com/generate"
+            },
+            "refreshUrl": {
+              "value": "https://daraja.onrender.com/generate"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
@johnmuchiriM and @Bkaraba for all our Independent Publisher we ask the iconBrandColor to be "#da3b01". can you please change it. please see out template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json).